content/pages/features.html

```diff
@@ -83,23 +83,48 @@ <h1 class="ui huge header">
   </div>
 </section>
 
-{% macro about_divide(text=None) %}
+<!-- First CTA -->
```

Comment:
Template comments are used through other pages instead of HTML comments. We want these comments for internal notes, they aren't meant to be part of the output.

```suggestion
{# First CTA #}
```